server/service/src/vaccination/get_vaccination_card.rs

```diff
@@ -122,7 +122,9 @@ pub fn get_suggested_date(
     };
 
     let suggested_date_by_min_interval = previous_dose.vaccination_date.and_then(|date| {
-        date.checked_add_signed(chrono::Duration::days(row.min_interval_days as i64))
+        date.checked_add_signed(chrono::Duration::days(
+            previous_dose.min_interval_days as i64,
+        ))
```

Comment:
Realised this was wrong! Per docs, min_interval on dose 1, should map to the number of days until dose 2 can be received. But we had it the other way, using the dose 2 min_interval 🙈